### ISSUE TITLE
Widget - make blue border on current card more subtle

### ISSUE THREAD

**Issue by inko9nito** [inko9nito/hpde#68]
The blue is too loud. It should be just anti darker than the current card background. Similar contrast ratio as white card and its gray border
